dev/tests/integration/testsuite/Magento/Sales/_files/guest_quote_with_addresses.php

```diff
@@ -10,12 +10,15 @@
 \Magento\TestFramework\Helper\Bootstrap::getInstance()->loadArea(\Magento\Framework\App\Area::AREA_FRONTEND);
 
 $objectManager = \Magento\TestFramework\Helper\Bootstrap::getObjectManager();
+$productRepository = $objectManager->create(\Magento\Catalog\Api\ProductRepositoryInterface::class);
```

Comment:
Please, use `get` for stateless services

Reply:
> Please, use get for stateless services
yes, I use 
```
$product = $productRepository->get('simple-product-guest-quote');
```
in line 34.

Reply:
I mean need to "GET" repository instead of creating.

Reply:
> I mean need to "GET" repository instead of creating it.

**fixed**

Thank you, @naydav !